arch/platform/cooja/contiki-conf.h

```diff
@@ -153,8 +153,8 @@ typedef unsigned long clock_time_t;
 /*---------------------------------------------------------------------------*/
 /* Virtual LED colors */
 #define LEDS_CONF_COUNT                  3
-#define LEDS_CONF_GREEEN                 1
-#define LEDS_CONF_RED                    2
-#define LEDS_CONF_YELLOW                 4
+#define LEDS_CONF_GREEN                 1
+#define LEDS_CONF_RED                    4
+#define LEDS_CONF_YELLOW                 2
```

Comment:
For code-style, maintain the indentation of the value? (and may I suggest moving `LEDS_CONF_RED` to the bottom for better readability